servicetalk-dns-discovery-netty/src/test/java/io/servicetalk/dns/discovery/netty/DnsServiceDiscovererObserverTest.java

```diff
@@ -62,6 +67,7 @@ class DnsServiceDiscovererObserverTest {
     private static final String SERVICE_NAME = "servicetalk";
     private static final String INVALID = "invalid.";
     private static final int DEFAULT_TTL = 1;
+    private static final String DISCOVERER_ID = "test";
```

Comment:
Consider using `getClass().getSimpleName()` to guarantee that each test class specifies its own id.